benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp/Makefile

```diff
@@ -15,6 +15,8 @@
 
 include ../common.mk
 
+MCXXFLAGS?="--Wn,-O3,-mtune=native,-finline-limit=50000,-fopt-info-all=gnu_opt_report.txt"
```

Comment:
Please could you add a comment to say what these flags are for (i.e. presumably they're for the OMPSs compiler?).

Reply:
They probably aren't needed - I put them there in case someone doesn't load the correct `compiler_setup` (or wants to use one where `MCXXFLAGS` isn't defined), but since they only work in the case that someone is using the gnu compiler anyway thats probably moot - I'll probably remove them.

Reply:
Removed in latest commits.